app/models/tag.rb

```diff
@@ -4,6 +4,8 @@ class Tag < ApplicationRecord
   virtual_has_one :category,       :class_name => "Classification"
   virtual_has_one :categorization, :class_name => "Hash"
 
+  has_one :container_label_tag_mapping # Only for tags assigned by ContainerLabelTagMapping.
```

Comment:
I just realized this is wrong, should be has_many — one may map multiple kinds of labels to one tag.

This is just the inverse relation to the mapping's `belongs_to :tag` from the mapping.  It has no effect on the actual schema - only the mapping table has a `tag_id`.

Yes, the reason it's useful is very unclear in this cut-down PR.  
When refreshing an entity, I should be able to clear auto-assigned tags that are no longer relevant (because label was changed/removed).  I can't wipe out _all_ tags, only ones I assigned.  Which are precisely tags figuring in at least one mapping tag_id.
Googling up on Rails, I discovered `Tag.joins(:custom_attribute_to_tag_mapping)` gives me them, but I must declare the association from Tag side for it to work.